### DISCUSSION TITLE
evaluate FPS of YOLOv4 on

### DISCUSSION THREAD

**Discussion by tututu11** [AlexeyAB/darknet#8433] [Q&A] [answered]
How many fps can your GPU reach, my GTX1080's AVG FPS is 20, is this normal? 
I used the method in readme.md. 
How much fps can you achieve with this method? can you name your fps and gpu models? 
Here is how to evaluate fps

How to evaluate FPS of YOLOv4 on GPU
Compile Darknet with GPU=1 CUDNN=1 CUDNN_HALF=1 OPENCV=1 in the Makefile
Download [yolov4.weights](https://github.com/AlexeyAB/darknet/releases/download/darknet_yolo_v3_optimal/yolov4.weights) file 245 MB: [yolov4.weights](https://drive.google.com/open?id=1cewMfusmPjYWbrnuJRuKhPMwRe_b9PaT) (Google-drive mirror yolov4.weights )
Get any .avi/.mp4 video file (preferably not more than 1920x1080 to avoid bottlenecks in CPU performance)
Run one of two commands and look at the AVG FPS:
include video_capturing + NMS + drawing_bboxes: ./darknet detector demo cfg/coco.data cfg/yolov4.cfg yolov4.weights test.mp4 -dont_show -ext_output
exclude video_capturing + NMS + drawing_bboxes: ./darknet detector demo cfg/coco.data cfg/yolov4.cfg yolov4.weights test.mp4 -benchmark

**Comment by stephanecharette** [comment, marked as answer]
Same thing I answered in your other ticket, issue #8430.

Some GPU results showing < 1.0 FPS to > 400 FPS can be seen here: https://www.ccoderun.ca/programming/2021-10-16_darknet_fps/

What you need to do to get results like this is in the FAQ:  https://www.ccoderun.ca/programming/darknet_faq/#fps
